README.md

```diff
@@ -55,20 +55,55 @@ Documentation for ert is located at [https://ert.readthedocs.io/en/latest/](http
 ## Developing
 
 *ERT* is Python and C software. To start developing, install it in editable
-mode:
+mode. We suggest using [virtual
+environment](https://docs.python.org/3/library/venv.html) to isolate the
+install (substitute the appropriate way of sourcing venv for your shell):
 
 ```
 $ git clone https://github.com/equinor/ert
 $ cd ert
+$ python -m venv venv
+$ . venv/bin/activate
+$ pip install --upgrade pip
 $ pip install -e .
 ```
 
+ERT requires a recent version of `pip` - hence we upgrade pip above. If your
+`pip` version is too old the installation of ERT will fail, and the error
+messages will be incomprehensible.
+
+### Trouble with setup
+
+If you encounter problems during install and attempt to fix them, it might be
+wise to delete the `_skbuild` folder before retrying an install.
+
 Additional development packages must be installed to run the test suite:
 ```
 $ pip install -r dev-requirements.txt
 $ pytest tests/
 ```
 
+As a simple test of your `ert` installation, you may try to run one of the
+examples, for instance:
+
+```
+$ cd test-data/local/poly_example
+$ # for non-gui trial run
+$ ert ensemble_experiment poly.ert
+$ # for gui trial run
```

Comment:
There's also `test_run` which is smaller and faster.

Reply:
could elaborate on this please? i don't see `test_run` as an example in `test-data/local`,  so I'm not sure what you're suggesting should be run precisely

Reply:
`ert test_run` is a subset of `ert ensemble_experiment` in that it only runs _one_ realization. So it's faster.

The `ensemble_experiment` will run an ensemble of 100 realizations, which is a bit heavy =)

Reply:
ah thanks i see now! gonna change that accordingly, and rebase to main, i think @pinkwah made some changes here in the meantime, too